### DISCUSSION TITLE
Problems with the tablet after the latest osu!(lazer) update.

### DISCUSSION THREAD

**Discussion by Bambi150240** [ppy/osu#28276] [Q&A] [answered]
After the update, the tablet (Gaomon s620) began to work strangely. The tablet is fully functional and works in both Windows and Osu! (default)

Video or screenshot showing the problem:
https://youtu.be/a4naiGeBP_g
[compressed-logs.zip](https://github.com/ppy/osu/files/15388407/compressed-logs.zip)

osu! version: Stable 20240513.1 (latest)

**Comment by peppy** [comment, marked as answer]
Please update once more.
